README.md

```diff
@@ -49,13 +51,92 @@ Now you may ask some questions like "What were our sales by year", and see how O
 
 See [Model Context Protocol Capability Overview](https://main.docs.pomerium.com/docs/capabilities/mcp) for more details.
 
+## Monitoring and Auditing MCP Tool Calls
```

Comment:
this breaks the "how does it work" section - can we move this block after the explanation of how token exchange and calls work ?